### PR TITLE
Kylox, Visionary Inventor + Alchemy cleanup

### PR DESCRIPTION
**Pull request by tool4ever**

### CODE REVIEW

File: forge-gui/res/cardsfolder/t/tahngarth_first_mate.txt [hunk 1]
```diff
@@ -5,7 +5,7 @@ PT:5/5
 S:Mode$ MinMaxBlocker | ValidCard$ Card.Self | Max$ 1 | Description$ CARDNAME can't be blocked by more than one creature.
 T:Mode$ AttackersDeclared | AttackingPlayer$ Player.Opponent | Execute$ TrigGainControl | TriggerZones$ Battlefield | OptionalDecider$ You | IsPresent$ Card.Self+tapped | TriggerDescription$ Whenever an opponent attacks with one or more creatures, if NICKNAME is tapped, you may have that opponent gain control of NICKNAME until end of combat. If you do, choose a player or planeswalker that opponent is attacking. NICKNAME is attacking that player or planeswalker.
 SVar:TrigGainControl:DB$ GainControl | Defined$ Self | NewController$ TriggeredAttackingPlayer | LoseControl$ EndOfCombat | RememberControlled$ True | SubAbility$ DBAttack
-SVar:DBAttack:DB$ ChangeCombatants | Defined$ Remembered | Attacking$ Player.Defending & Valid Planeswalker.Defending | SubAbility$ DBCleanup
+SVar:DBAttack:DB$ ChangeCombatants | Defined$ Remembered | Attacking$ .Defending & Valid Planeswalker.Defending | SubAbility$ DBCleanup
```
Comment (Northmoc) [inline forge-gui/res/cardsfolder/t/tahngarth_first_mate.txt:8]: This `Attacking$` value seems weird to me... the leading `.` looks like a typo

File: forge-gui/res/cardsfolder/upcoming/kylox_visionary_inventor.txt [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+K:Ward:2
+K:Haste
+T:Mode$ Attacks | ValidCard$ Card.Self | Execute$ TrigSac | TriggerZones$ Battlefield | TriggerDescription$ Whenever CARDNAME attacks, sacrifice any number of other creatures, then exile the top X cards of your library, where X is their total power. You may cast any number of instant and/or sorcery spells from among the exiled cards without paying their mana costs.
+SVar:TrigSac:DB$ Sacrifice | Defined$ You | Amount$ SacX | SacValid$ Creature.Other | RememberSacrificed$ True | Optional$ True | SubAbility$ DBDig
+SVar:DBDig:DB$ Dig | Defined$ You | DigNum$ X | ChangeNum$ All | DestinationZone$ Exile | Imprint$ True | SubAbility$ DBPlay
+SVar:DBPlay:DB$ Play | Valid$ Card.IsImprinted+YouOwn | ValidZone$ Exile | ValidSA$ Instant,Sorcery | Controller$ You | WithoutManaCost$ True | Optional$ True | Amount$ All | SubAbility$ DBCleanup
+SVar:DBCleanup:DB$ Cleanup | ClearRemembered$ True | ClearImprinted$ True
+SVar:X:RememberedLKI$CardPower
+SVar:SacX:Count$Valid Creature.Other
+AI:RemoveDeck:All
```
Comment (Northmoc) [inline forge-gui/res/cardsfolder/upcoming/kylox_visionary_inventor.txt:15]: add `DeckHas:Ability$Sacrifice` and probably `DeckHints:Type$Instant|Sorcery` ?

File: forge-gui/res/cardsfolder/t/tahngarth_first_mate.txt (hunk 1 above)
Comment (tool4ever) [reply forge-gui/res/cardsfolder/t/tahngarth_first_mate.txt:8]: This is the correct syntax for now 🤹‍♂️

**Review by Northmoc — APPROVED** [review]
Looks good to me...

Thanks for cleaning up my mess on Lupine Harbingers 😅

**Pull request merged**